### PR TITLE
mkosi: Add basic editors to final images

### PR DESCRIPTION
**Pull request by daandemeyer**
It's often useful to have an editor available to edit some random
config file in the final image. Let's install some basic editors
that don't take up too much space.

### CODE REVIEW

File: .mkosi/mkosi.debian [hunk 1]
```diff
@@ -73,7 +73,9 @@ Packages=
         # We pull in the -dev package here, since the binary ones appear to change names too often, and the -dev package pulls the right deps in automatically
         libtss2-dev
         locales
+        nano
         strace
+        vi
```
Comment (bluca) [inline .mkosi/mkosi.debian]: it's ```vim``` in Debian/Ubuntu

Comment (daandemeyer) [reply .mkosi/mkosi.debian]: The entirety of vim is quite a bit bigger than the vi package (at least on Arch) but it doesn't seem like a separate vi is available on Ubuntu/Debian so let's go wirh vim.

**Comment by vcaputo** [comment]
On debian there's nvi, and vim-tiny.

nvi is much smaller and strives to replicate OG BSD vi.

**Comment by bluca** [comment]
> On debian there's nvi, and vim-tiny.
> 
> nvi is much smaller and strives to replicate OG BSD vi.

good point, vim-tiny seems to be the right one (same source but minimized and with no graphical deps)

**Comment by DaanDeMeyer** [comment]
Adjusted to vim-tiny

**Review by bluca — APPROVED** [review]

**Pull request merged**